tabulator/stream.py

```diff
@@ -444,8 +445,37 @@ def builtin_processor(extended_rows):
 
                 yield (row_number, headers, row)
 
+        def skip_negative_rows(extended_rows):
+            """
+            This processor will skip rows which counts from the end, e.g.
+            -1: skip last row, -2: skip pre-last row, etc.
+            Rows to skip are taken from  Stream.__skip_rows_by_numbers
+            """
+            rows_to_skip = [n for n in self.__skip_rows_by_numbers if n < 0]
+            if not rows_to_skip:
+                for row in extended_rows:
+                    yield row
+            else:
+                buffer_size = abs(min(rows_to_skip)) + 1
```

Comment:
This is a bit confusing - the buffer size if actually `abs(min(rows_to_skip))` (without the `+1`)

Reply:
Agree, will fix.